src/types.ts

```diff
@@ -52,3 +52,95 @@ export interface AddedRemovedUpdate {
     removed: APIOrderWithMetaData[];
     updated: APIOrderWithMetaData[];
 }
+
+// Staking types
+export interface RawEpoch {
+    epoch_id: string;
+    starting_transaction_hash: string;
+    starting_block_number: string;
+    starting_transaction_index?: string;
+    starting_block_timestamp?: string;
+    ending_transaction_hash?: null;
+    ending_transaction_index?: null;
+    ending_block_number?: null;
+    ending_block_timestamp?: null;
+}
+
+export interface TransactionDate {
+    blockNumber: number;
+    txHash: string;
+    timestamp?: string;
+}
+
+export interface Epoch {
+    epochId: number;
+    epochStart: TransactionDate;
+}
+
+export interface RawPool {
+    pool_id: string;
+    operator: string;
+    created_at_block_number: string;
+    created_at_transaction_hash: string;
+    created_at_transaction_index: string;
+    maker_addresses: string[];
+    verified?: string;
+    logo_url?: string;
+    location?: string;
+    bio?: string;
+    website?: string;
+    name?: string;
+}
+
+export interface PoolMetadata {
+    isVerified: boolean;
+    logoUrl?: string;
+    location?: string;
+    bio?: string;
+    websiteUrl?: string;
+    name?: string;
+}
+
+export interface Pool {
+    poolId: number;
```

Comment:
Not a fun of this being specified as a number. The sequence is not meaningful.

Reply:
Not sure what you mean.